src/main/ank/2.3-free-monoids.md

```diff
@@ -0,0 +1,91 @@
+```Haskell
+class Monoid m where
+  mempty :: m
+  mappend :: m -> m -> m
+```
+```kotlin:ank:silent
+interface Monoid<A> {
+  fun empty(): A
+  fun combine(a: A, b: A): A
+}
+```
+................
+```Haskell
+instance Monoid [a] where
+  mempty = []
+  mappend = (++)
+```
+```kotlin:ank:playground
+import arrow.extension
+import arrow.typeclasses.MonoidOf
+
+@extension 
+interface Monoid<A>: MonoidOf<List<A>> {
```

Comment:
```suggestion
interface Monoid<A>: MonoidOf<ForListK> {
```

Reply:
That shouldn't be the case.

Reply:
I thought ...Of<HKT> expects a HKT?
At least, that is what I've been said.

Reply:
Monoid is aggregation of regular values, check the existing implementation of `ListK`.